examples/petclinic/service.bal

```diff
@@ -0,0 +1,465 @@
+import ballerina/http;
+import petclinic.db;
+import ballerina/log;
+import ballerina/persist;
+import ballerina/sql;
+
+configurable ServerConfig serverConfig = ?;
+
+// HTTP listener
+listener http:Listener httpListener = new (serverConfig.port, config = {
```

Comment:
We only need `port` to be configurable but better to have a default value to make it as an optional configuration